projects/kit/components/checkbox/checkbox.component.ts

```diff
@@ -56,13 +50,12 @@ export class TuiCheckboxComponent
         @Self()
         @Inject(NgControl)
         control: NgControl | null,
-        @Optional()
-        @Inject(TUI_CHECKBOX_DEFAULT_OPTIONS)
-        options: CheckboxOptions | null,
+        @Inject(TUI_CHECKBOX_OPTIONS)
+        options: CheckboxOptions,
```

Comment:
let's just make it `private readonly` and write `size: TuiSizeL = this.options.size`